apps/io-wallet-user-func/src/app/main.ts

```diff
@@ -88,6 +95,7 @@ app.http("createWalletInstance", {
   handler: CreateWalletInstanceFunction({
     attestationServiceConfiguration: config.attestationService,
     nonceRepository,
+    telemetryClient: appInsightsClient,
```

Comment:
it's ok as is, but keep in mind that there's no need to pass the `defaultClient`. you can import it directly from the appinsighrts module as it's a singleton